examples/v2/subscriber/subscriber.nim

```diff
@@ -0,0 +1,73 @@
+import
+  confutils,
+  std/[tables,sequtils],
+  chronicles,
+  chronicles/topics_registry,
+  chronos,
+  stew/shims/net,
+  libp2p/crypto/crypto,
+  eth/keys,
+  eth/p2p/discoveryv5/enr
```

Comment:
do you fix the imports manually? or do we have some linter?

Reply:
What is the pattern here @LNSD ? 🤔 I usually have `std/` first, then external imports alphabetically, then internal, relative imports from furthest removed to closest. (Not really established before, but happy to make this convention.) I thought it's similar here, but then I'm unsure about `stew` 🤷

Reply:
> do you fix the imports manually? or do we have some linter?

This is completely manual. There are no decent code formatters in nim.

> What is the pattern here @LNSD ?

`import`
1. `std/[<import1>, <import2>]
2. `stew/<import1>
3. `stew/<import2>
4. `chronicles`
5. `chronos`
6. The rest of the imports.
`import`
1. Local modules ordered alphabetically (e.g., first `node` modules, then `protocol` modules, then `utils` modules)

Avoid grouping them by using brackets since it makes it harder to find and replace to upgrade things. `std` library is the exception since, AFAIK, there are no nested modules.

`stew` is a "special" module. It is our override of many standard library imports. So I put it next to the `std` module at the top.

Reply:
Cool. Let's add to conventions?